supply-chain-security/compromises/2019/purescript-npm.md

```diff
@@ -0,0 +1,12 @@
+# Malicious code in purescript npm installer dependencies
+Malicious code was inserted in dependencies of the purescript npm installer. 
+The code was inserted in the packages load-from-cwd-or-npm and rate-map.
+
+
+## Impact
+Both packages have ~ 1400 downloads weekly.
```

Comment:
Can we elaborate a bit more on timeframe here? "Malicious packages may have been downloaded X times, over Y time?" (assuming we count from the discovery/rectification date).